Dockerfile

```diff
@@ -1,42 +1,82 @@
-FROM golang:1.17-alpine as cloudreve_builder
+# 多阶段构建: 编译发布物
+# 注意: 该阶段除最终发布物外不会影响最终镜像体积, 删除缓存、减少包安装等无意义
+FROM golang:1-alpine AS builder
```

Comment:
I think specify the golang version with 1.17 would be better, or you can write down why you using this image